ros-kinetic/moveit_core/moveit_core-0.9.11.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright 2018 Open Source Robotics Foundation
 # Distributed under the terms of the BSD license
 
-EAPI=6
+EAPI=5
```

Comment:
Is there any particular reason this needs to change to EAPI 5?

This ebuild was autogenerared by Superflore -- I would strongly prefer you follow the contribution guide for those ebuilds (since this ebuild will be updated in the future automatically, and I cannot guarantee the patch you added here will persist).

Reply:
(also, Superflore now has an ebuild: `dev-python/superflore`)

Reply:
EAPI 6 implies patch -p1, so I updated the patch now to work this way. See updated branch